### PR TITLE
added Unenroll button

### PR DESCRIPTION
**Pull request by aiyer786**

### CODE REVIEW

File: frontend/src/components/UserCaloriesPage.js [hunk 1]
```diff
@@ -243,6 +245,55 @@ function UserCaloriesPage(props) {
...
+      Authorization: "Bearer " + props.state.token,
+    }
+  })
+  .then(response => {
+    setEnrolledEvents(response.data); 
+  })
+  .catch(error => {
+    console.error("Error fetching enrolled events:", error);
+  });
+}, []);
```
Comment (shashank-madan) [inline frontend/src/components/UserCaloriesPage.js:270]: Please delete this. The expected functionality is already being done. The route specified is not mapped in the backend as well, rendering this code-block unused.

Comment (aiyer786) [reply frontend/src/components/UserCaloriesPage.js]: Deleted the redundant code.

File: frontend/src/components/UserCaloriesPage.js [hunk 2]
```diff
@@ -453,6 +477,13 @@ const redirectToEventWithModalOpen = (eventName) => {
```
Comment (shashank-madan) [inline frontend/src/components/UserCaloriesPage.js:477]: Unnecessary whitespace

**Review by shashank-madan — APPROVED** [review]
